index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html5>
+<html>
+<head>
+    <title>HTML/CSS Quiz</title>
+    <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+<article id="article 1" class="article">
+    <header class="main_header">
+        <h1>Coors Light</h1>
+    </header>
+    <section class="section1">
+        <div class="image"><img src="https://unsplash.it/200/?random=1"></div>
+        <div class="prod_descrip">Davy Byrne’s metemdivsychosis Molly mellow like a shot off a shovel Bloom. La ci darem la mano stately seedcake Sandycove 7 Eccles Street Molly smellow kidneys moody brooding. Sirens Ithaca Buck Mulligan Kinch gorgonzola Rudy song. La ci darem la mano Nausicaa Tinbad the Tailor fortyfoot ben Bloom Elijah ineluctable modality of the visible Dedalus melons Ithaca 7 Eccles Street letter transmigration oxen of the sun divenelodive. </div>
+        <footer>Always Available</footer>
+    </section>  
```

Comment:
Be aware of whitespace left in your files. Inevitably it will cause headaches in the form of merge conflicts. The best way to stem it is to either download the trailing spaces package for sublime to highlight your whitespace or add a line in your user config preferences to "trim_trailing_white_space_on_save": true.